de-lib/src/main/java/org/iplantc/de/client/services/impl/AnalysisServiceFacadeImpl.java

```diff
@@ -317,5 +317,20 @@ public void getVICELogs(String id,
 
     }
 
+    @Override
+    public void getViceTimeLimit(String id, DECallback<String> callback) {
+        String address = ANALYSES + "/" + id + "/time-limit";
+        ServiceCallWrapper wrapper = new ServiceCallWrapper(GET, address);
+        deServiceFacade.getServiceData(wrapper, callback);
+    }
+
+    @Override
+    public void setViceTimeLimit(String id, DECallback<String> callback) {
```

Comment:
Not sure about naming this method `setViceTimeLimit`, since it doesn't actually accept a time limit value to set with this endpoint call 🤔 

Maybe `extendViceTimeLimit` is more accurate?